clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/EhcacheMessageType.java

```diff
@@ -61,7 +60,6 @@
   public static final String MESSAGE_TYPE_FIELD_NAME = "opCode";
   public static final int MESSAGE_TYPE_FIELD_INDEX = 10;
   public static final EnumMapping<EhcacheMessageType> EHCACHE_MESSAGE_TYPES_ENUM_MAPPING = newEnumMappingBuilder(EhcacheMessageType.class)
-    .mapping(CONFIGURE, 1)
```

Comment:
Since we'd be breaking the comms protocol anyway we could correct all these index values at the same time right?